DEPRECATED_delius-po-test2/sub-projects/spg.tfvars

```diff
@@ -71,7 +71,7 @@ spg_iso_asg_max = 2
 spg_iso_asg_min = 1
 spg_iso_service_desired_count = 1
 spg_iso_ecs_memory = 1881
```

Comment:
shouldn't this be higher than the JAVA MAX?

Reply:
applied same settings as POtest1 and is now resolved